python/lsst/ctrl/pool/parallel.py

```diff
@@ -15,7 +15,6 @@
 import contextlib
 from lsst.pipe.base import CmdLineTask, TaskRunner
 from .pool import startPool, Pool, NODE, abortOnError, setBatchType
-from . import log  # register pickle functions for log
```

Comment:
I didn't see this line in parallel.py? Not sure where it went. Added it back in as "from . import log # noqa

Reply:
Also - is this import unnecessary? I get this:

================================================================ FAILURES =================================================================
________________________________________ FLAKE8-check(ignoring E133 E226 E228 E251 N802 N803 N806) ________________________________________
/home/jcarlin/repos/ctrl_pool/python/lsst/ctrl/pool/parallel.py:64:5: F811 redefinition of unused 'log' from line 15

Reply:
Adopted your suggestion. (Not sure I understand why that import is needed, but trust your wisdom.)

Reply:
it's needed because there is code in ctrl.pool.log that needs to be run even if you don't use it directly (it enables pickling). Not sure why that couldn't be in the `__init__.py` though.